glossary.md

```diff
@@ -103,15 +103,30 @@ A file with the `.hpp` or `.h` file extension.
 # Node.js & C++
 
 #### Node
+
+A command line tool that consists of a set of bindings to V8 Javascript. These bindings allow you to use Javascript for implementing "lower-level" operations like working with the file system, threads, and scripting servers. More specifically, Node allows you to interact with the [POSIX api](https://github.com/mapbox/cpp/blob/master/glossary.md#posix), using Javascript.
+
+Node is made up of a combination of parts:
+- V8: to interpret Javscript
+- Threadpool ([libeio](http://software.schmorp.de/pkg/libeio.html)): To do file I/O or any other blocking system call
+- Event Loop ([libev](http://software.schmorp.de/pkg/libev.html)): A main process (or main thread) that responds to an event queue: grabs the top item in the event queue, executes it, and then grabs the next item.
+- C++ Node bindings: Expose the low-level interface to non-blocking POSIX calls
+- Thread and Event Loop Manager ([libuv](http://libuv.org/)): This library contains libeio and libev and helps manage them.
+
```

Comment:
Libuv reimplemented but does not contain libeio and libev. Those were inspirations but did not run on windows, so libuv replaces them.